packages/contracts/test/ManagedService.t.sol

```diff
@@ -209,6 +209,6 @@ contract ManagedService_Test is Test, ManagedService {
 
     function test_RevertIfOwnerIsAddressZero() external {
         vm.expectRevert("ManagedService: admin cannot be address(0)");
-        new ManagedService{ salt: keccak256("1") }(address(0));
```

Comment:
The update to Foundry to use the create2 factory in the simulation caused this test to break since the error if the create2 factory is used differs from the error we expect on line 211. I resolved this issue just by deploying via the normal create opcode. 

For reference, this is the result if the create2 factory is used:
<img width="550" alt="Screenshot 2024-04-18 at 3 55 21 PM" src="https://github.com/sphinx-labs/sphinx/assets/25066232/4f38c567-b3ca-4d1f-86d9-eab9ed5d936d">